_posts/2021-03-09-gsoc2021.md

```diff
@@ -0,0 +1,118 @@
+| 2       | [AtomGroupGroup and Rings]({{ site.github.wiki }}/Project-Ideas-2021#project-2-atomgroupgroup-and-rings)                           | medium     | Write analysis module for pi stacking analysis                                                 | Python                      | @richardjgowers, @micaela-matta, @IAlibay        |
+| 3       | [Generalise Groups]({{ site.github.wiki }}/Project-Ideas-2021#project-3-generalise-groups)                                         | medium     | Generalise concept of groups                                                                   | Python                      | @lilyminium, @fiona-naughton (?) |
+| 4       | [Cythonisation of AtomGroup]({{ site.github.wiki }}/Project-Ideas-2021#project-4-cythonise-atomgroup)                              | easy      | Cythonise AtomGroup for use with C/++                                                          | Python, Cython, C/++        | @richardjgowers, @hmacdope, @lilyminium (maybe)|
+| 5       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/Project-Ideas-2021#project-5-extend-interoperability)                  | medium     | Extend converters module to other relevant packages                                            | Python                      | @lilyminium, @IAlibay  |
+| 6       | [General unit cell representation]({{ site.github.wiki }}/Project-Ideas-2021#project-6-general-unit-cell-representation)                  | easy     |  change unit cell representation to keep track of box rotations and so improve analysis of simulations under periodic boundary conditions | Python                      | @orbeckst  |
+
+# Information for Students
+
+You must meet our [own requirements] if you want to be a student with MDAnalysis
+this year (read all the docs behind these links!). You must also meet the
+[eligibility criteria]. Our [GSoC FAQ][] collects common questions
+from students.
+
+The MDAnalysis community values diversity and is committed to
+providing a productive, harassment-free to every member. Our [Code of
+Conduct] explains the values that we as a community uphold. *Every
+member (and every GSoC student) agrees to follow the [Code of
+Conduct]*.
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
+## Complete the Quick Start Guide
+
+We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+## Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us what you plan to work
+on during the summer or what you have already done with MDAnalysis.
+
+## Close an issue of MDAnalysis
+
+You must have *at least one commit in the development branch of
```

Comment:
Do we want to be a bit more strict on what we mean by a commit here? I'm always worried that folks will just fix a small amount of docstring and then walk away thinking they did the minimum.

Reply:
People did it in the past. I don't know how to specify what a sizable commit is. Ultimately we can decide not to merge or ask for more work. When it comes to choosing students then we always take their level of engagement with us into account. I'd say let them show how much they _want_ to engage instead of training them to pass a test. 

I would leave it as it is but I am happy to go the other way, too.